alica_engine/include/engine/containers/AgentAnnouncement.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "engine/AgentIDConstPtr.h"
+
+#include <string>
+#include <tuple>
+#include <vector>
+
+namespace alica
+{
+struct AgentAnnouncement
+{
+    using CapabilityPair = std::pair<std::string, std::string>;
+    AgentAnnouncement()
+            : senderID(nullptr)
+    {
+    }
+    AgentIDConstPtr senderID;
+    uint32_t token;
```

Comment:
Is it really better to add 4 byte randomness to detect duplicates instead of simply increasing the size of the id coresspondingly to 12 bytes?